openpype/hosts/blender/plugins/publish/extract_playblast.py

```diff
@@ -26,6 +26,11 @@ class ExtractPlayblast(Extractor):
     def process(self, instance):
         self.log.info("Extracting capture..")
 
+        # Stop if errors to avoid useless render
+        for result in instance.context.data["results"]:
+            if not result["success"]:
+                return
```

Comment:
I think this should raise an error or at least a warning since the extractor has been skipped. Even though the end result is the same, I don't think it should be displayed as successful.

Reply:
you're right, changed